### PR TITLE
[WIP] Ens name search 

### PR DESCRIPTION
**Pull request by james-prado**
Closes https://github.com/MyEtherWallet/MyEtherWallet/issues/87

- Add ENS search bar
- Add Components for Name Open, Name Taken, etc... [details here](https://docs.ens.domains/en/latest/userguide.html#starting-an-auction)
- Add `<BidModal />`for placing bids

Todo: 
- fix negative margin styles on mobile screen widths
- add tests for ENS lib
- validation for user forms to place bids
- generate bid transactions

#### Blocking:
I’ve got all the user input I need to generate transactions to place a bid, and all the UI is ready, I just don’t know how to generate a transaction to place a bid, or how to get any data I need from that transaction, like the bid reveal date

tldr: How do I encode all the user data into a transaction, and then decode the information to display back to the user?

### CODE REVIEW

**Comment by james-prado** [comment]
Stale branch, closing.
